component/pre.jsonnet

```diff
@@ -0,0 +1,52 @@
+/*
+    This Job ensures smooth upgrade of Stackgres from version > 1.5.
+    It solves issues with breaking changes introduced in helm charts, as well as egg-chicken (certs, issuers, secrets) issues during upgrade.
+*/
+
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local kube = import 'lib/kube.libjsonnet';
+local params = inv.parameters.stackgres_operator;
+local instance = inv.parameters._instance;
+
+
+local CleanCertsJobName = 'clean-certificates-job';
+
+local job = kube.Job('') {
+  metadata: {
+    generateName: CleanCertsJobName + '-',
+    annotations: {
+      'argocd.argoproj.io/hook': 'PreSync',
+    },
+  },
+  spec: {
+    template: {
+      spec: {
+        containers: [
+          {
+            args: [],
+            command: [
+              '/bin/bash',
+              '-cex',
+              importstr 'scripts/preCleanCerts.sh',
+            ],
+            env+: [
+              {
+                name: 'NAMESPACE',
+                value: params.namespace,
+              },
+            ],
+            image: 'docker.io/bitnami/kubectl:1.24',
```

Comment:
Don't hardcode images. We might run into docker.io rate limit and want to be able to override this somewhere.

Reply:
fixed buy using params.images.registry also switched to appuio `oc` image supported by us because I couldn't find kubectl image anywhere on quay or ghcr where it was from trusted provider